invenio_app_rdm/upgrade_scripts/migrate_13_0_to_14_0.py

```diff
@@ -0,0 +1,173 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2025 CERN.
+#
+# Invenio-App-RDM is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Record migration script from InvenioRDM 13.0 to 14.0.
+
+Disclaimer: This script is intended to be executed *only once*, namely when
+upgrading from InvenioRDM 13.0 to 14.0!
+If this script is executed at any other time, probably the best case scenario
+is that nothing happens!
+
+
+This script has been tested with following data:
+
+- record
+    - internal_notes
+"""
+
+from click import secho
+from invenio_access.permissions import system_identity
+from invenio_rdm_records.proxies import current_rdm_records_service as records_service
+from invenio_search.engine import dsl
+
+
+def run_upgrade(has, migrate_record, migrate_draft):
+    """Run upgrade."""
+    # Handle published records
+    published_records = records_service.search(
+        system_identity,
+        params={"allversions": True, "include_deleted": True},
+        extra_filter=has,
+    )
+    for result in published_records.hits:
+        try:
+            migrate_record(result)
+        except Exception as error:
+            secho(f"> Error {repr(error)}", fg="red")
+            error = f"Record {result['id']} failed to update"
+
+    # Handle draft records
+    draft_records = records_service.search_drafts(
+        system_identity,
+        params={"allversions": True},
+        extra_filter=has,
+    )
+    for result in draft_records.hits:
+        try:
+            migrate_draft(result)
+        except Exception as error:
+            secho(f"> Error {repr(error)}", fg="red")
+            error = f"Draft {result['id']} failed to update"
+
+
+def run_update_for_resource_type():
+    """Run update for resource type."""
+
+    def migrate_resource_type_in_record(hit_result):
+        """
+        Update resource type from event to publication-dissertation.
+        We go through the service layer to automatically trigger the DOI update and re-indexing.
+        """
+        print(f"Updating resource type for {hit_result['id']}")
+        record = records_service.read_latest(system_identity, hit_result["id"])
+        if record.data["metadata"]["resource_type"]["id"] != "event":
```

Comment:
Why `event`?

Reply:
forgot to change after testing, fixed now!